src/Components/WorkExpereince.js

```diff
@@ -0,0 +1,293 @@
+import React, { useEffect } from "react";
+import { Container } from "react-bootstrap";
+import Col from "react-bootstrap/Col";
+import Nav from "react-bootstrap/Nav";
+import Row from "react-bootstrap/Row";
+import Tab from "react-bootstrap/Tab";
+import Tabs from "react-bootstrap/Tabs";
+import { useMediaQuery } from "react-responsive";
+import aos from "aos";
+import "aos/dist/aos.css";
+
+function Work() {
+  useEffect(() => {
+    aos.init({ duration: 1500 });
+  }, []);
+  const isMobile = useMediaQuery({ query: "(max-width: 720px)" });
+  let element;
+  if (isMobile) {
+    element = <Mobile />;
+  } else {
+    element = <PC />;
+  }
+  return element;
```

Comment:
by the way, this kind of pattern reads nicer like so, if you ever want to use it again:

```suggestion
  return isMobile ? <Mobile /> : <PC />
```